Prometheus/package.json

```diff
@@ -11,26 +11,32 @@
   },
   "dependencies": {
     "@module-federation/enhanced": "^0.8.9",
-    "@prometheus-io/codemirror-promql": "^0.301.0",
-    "color-hash": "^2.0.2"
+    "@nexucis/fuzzy": "^0.5.1",
+    "@prometheus-io/codemirror-promql": "^0.43.0",
+    "color-hash": "^2.0.2",
+    "qs": "^6.13.0"
   },
   "peerDependencies": {
     "@emotion/react": "^11.7.1",
     "@emotion/styled": "^11.6.0",
     "@hookform/resolvers": "^3.2.0",
     "@perses-dev/components": "^0.50.1",
     "@perses-dev/core": "^0.50.1",
+    "@perses-dev/dashboards": "^0.50.1",
```

Comment:
I guess ideally explorer shouldn't be dependent on the package `@perses-dev/dashboards` that sounds heavy.

WDYT @jgbernalp @celian-garcia @Gladorme ?

Reply:
Yes is heavy. The main component used here is the Panel, which we might move to @perses-dev/components. I Just wanted to do minimal changes so this can work.

Reply:
For the moment this will be provided by Perses UI as a shared dependency